### DISCUSSION TITLE
Is the theme in GNOME Files, et al, supposed to match the look theme?

### DISCUSSION THREAD

**Discussion by jc00ke** [regolith-linux/regolith-desktop#799] [General]
![image](https://user-images.githubusercontent.com/18191/209602542-bf103280-84a2-4670-8d23-38e6f946c068.png)

The only way I can change this is to pick "Dark" in the control center, but then the background is black, not the dark blue I'd expect from Solarized Dark.

![image](https://user-images.githubusercontent.com/18191/209602724-cdbd24a1-4748-46b6-bd0e-1aa4435d524f.png)

**Comment by kgilmer** [comment]
Yes, I've noticed this as well.  I haven't dug into this issue yet, but am guessing that GTK4 has codified some of their earlier policies regarding theming (meaning; you can't) but am not sure.  I will eventually have time to look into this but if [changing the gtk theme via gsettings](https://askubuntu.com/questions/262868/how-to-set-icons-and-theme-from-terminal) has no effect (it doesn't but you may want to verify), then we either need some new way of specifying the theme or we simply must let GNOME be GNOME :shrug:

**Reply by peterrus** [reply]
> but am guessing that GTK4 has codified some of their earlier policies regarding theming (meaning; you can't) but am not sure.

It seems that way, yes.

https://old.reddit.com/r/gnome/comments/18rkzrx/gtk4_stylesheets_for_global_specific_theming/kf200ck/
